src/dmangle.d

```diff
@@ -630,17 +629,17 @@ public:
          * -0X1.1BC18BA997B95P+79   => N11BC18BA997B95P79
          * 0X1.9P+2                 => 19P2
          */
-        if (Port.isNan(value))
+        if (isNaN(value))
```

Comment:
Where is it getting `isNaN` from? Since there are many `isNan` functions, and `isInfinity` functions, I'd prefer to see this name qualified.